### PR TITLE
Add SelectHandleDropdown component and its story

### PR DESCRIPTION
**Pull request by Dauntless-Jacobson**

### CODE REVIEW

**Comment by Dauntless-Jacobson** [comment]
dropdown

**Comment by Dauntless-Jacobson** [comment]
dropdownHandle args

File: src/web-components/DropdownButton/index.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ *   <div slot="slottedDropdown"></div>
+ * </dropdown-button>
+ * ```
+ */
+
+@customElement('dropdown-button')
+export class DropdownButton extends LitElement {
+    @state() dropdownOpen = false;
+
+    @property({ type: Array }) options: string[] = [];
```
Comment (bigirishlion) [inline src/web-components/DropdownButton/index.ts:36]: Let's remove if not being used

**Review by bigirishlion — CHANGES_REQUESTED** [review]
✌️

**Review by bigirishlion — APPROVED** [review]
🧇

**Pull request merged**